src/main/scala/com/github/unchama/seichiassist/subsystems/vote/subsystems/fairy/bukkit/actions/BukkitRecoveryMana.scala

```diff
@@ -18,223 +16,219 @@ import com.github.unchama.seichiassist.subsystems.vote.subsystems.fairy.domain.p
   FairyManaRecoveryState,
   FairyUsingState
 }
-import com.github.unchama.targetedeffect.{SequentialEffect, UnfocusedEffect}
+import com.github.unchama.seichiassist.{MineStackObjectList, SeichiAssist}
+import com.github.unchama.targetedeffect.SequentialEffect
 import com.github.unchama.targetedeffect.commandsender.MessageEffect
 import org.bukkit.ChatColor._
 import org.bukkit.entity.Player
 
 import java.time.LocalDateTime
+import java.util.UUID
 import scala.util.Random
 
-object BukkitRecoveryMana {
+class BukkitRecoveryMana[F[_]: ConcurrentEffect: Applicative, G[_]: ContextCoercion[*[_], F]](
```

Comment:
```suggestion
class BukkitRecoveryMana[F[_]: ConcurrentEffect, G[_]: ContextCoercion[*[_], F]](
```

`ConcurrentEffect`は`MonadError`を満たすので`Applicative`は冗長です

参考:
[![cats typeclass hierarchy](https://cdn.jsdelivr.net/gh/tpolecat/cats-infographic@master/cats.svg)](https://typelevel.org/cats/typeclasses.html)